helm/cas-bciers/templates/cronJobs/check-file-status.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: check-document-file-status
+  labels: {{- include "cas-bciers.labels" . | nindent 4 }}
+    component: backend
+  namespace: {{ .Release.Namespace }}
+spec:
+  suspend: false
+  schedule: "*/1  * * * *"
```

Comment:
Every minute might be too often - unless we have a mechanism to check whether there is already another instance running. I'd suggest doing that, this way if a bug happens, we don't create a bunch of pods

Reply:
We could use a tool like `flock` to do that, this will just require a small PVC mounted on this pod? https://man7.org/linux/man-pages/man1/flock.1.html

Reply:
@pbastia I've updated the helm charts with d0872bd to add a `flock` using a tiny 10mb PVC mount.